enh(mc): automatic commit for network-cisco-wap-snmp (#3117)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-cisco-wap-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ## Pack assets

Author: ```suggestion
Cisco  is a provider of network solutions.

The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of the Cisco Wireless Access Point equipment.

## Pack assets
```

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Cisco  is a provider of network solutions.

> The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of the Cisco Wireless Access Point equipment.

Reviewer: ```suggestion
The Centreon Monitoring Connector *Cisco Wap SNMP* relies on the SNMP protocol to query and collect the status and metrics of Cisco Wireless Access Point devices.
```

> The SNMP agent must be configured and running. Please refer to the manufacturer documentation to achieve this.

Reviewer: 1. I think this is not the standard text, is it?
2. FR and EN are different

> The SNMP agent must be configured and running. Please refer to the manufacturer documentation to achieve this.

Author: ```suggestion
The SNMP service must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

```
